web/templates/pages/member_page.html

```diff
@@ -0,0 +1,45 @@
+{% extends 'base.html' %}
+
+{% load staticfiles %}
+
+{% block title %}{{ user.fullname }}{% endblock %}
+
+{% block content%}
+
+	<div class="row">
+		<div class="member_profile">
+			<div class="col-md-12">
+			<h2>{% if user.fullname %}
+			{{ user.fullname }} 
+			{% else %}
+			{{ user.username }}
+			{% endif %}'s profile</h2>
+			{% for field in form %}
+			{{ field.error }}
+			{% endfor %}
+			</div>	
+		</div>
+	</div>
+	<div class="row">
+		<div class="col-md-4 col-xs-6">
+			<!--<img  src="{% static 'img/{{user.profile_picture}}' %}" class="img-responsive" alt="Responsive image">-->
+			<img src="../../static/img/{{user.profile_picture}}">
```

Comment:
do we really need these relative paths? doesn't "static" work (as in the commented out line directly above)

also, if you're replacing code (as you did in this case), remove the commented out code